### PR TITLE
Update inquiry order notes to use inquiry specific content

### LINKED ISSUE #9612: Order notes says "payment has been disputed" for inquiries
### Describe the bug

When a payment inquiry occurs, an order note shows _"Payment has been disputed for `amount` with reason `reason`. Response due by `date`."_

### To Reproduce

1. Purchase an order with the test card `4000000000001976`
2. View the order in WooCommerce Admin.
3. See order note mentions "dispute"

### Actual behavior

A note with the following text is added to the **Order notes**: _"Payment has been disputed for `amount` with reason `reason`. Response due by `date`."_

### Screenshots

Image

### Expected behavior

The note should not indicate a dispute, instead a payment inquiry should be noted. 

### Desktop (please complete the following information):

* OS: macOS 15.0.1
* Browser: Google Chrome
* Version: 129.0.6668.101

### Additional context

Discussed in detail in Slack [here](https://a8c.slack.com/archives/C7U3Y3VMY/p1729257283105179). 

**Note: There might be additional enhancements needed to the UX and the way we display inquiries.**

### PR DESCRIPTION
**Pull request by brucealdridge**
Fixes #9612

#### Changes proposed in this Pull Request

Update order notes copy to reference "inquiries" rather than "disputes" when a inquiry occurs.

New Dispute: `Payment has been disputed for %1$s with reason "%2$s". <a>Response due by %3$s</a>.`
New Inquiry: `A payment inquiry has been raised for %1$s with reason "%2$s". <a>Response due by %3$s</a>.`

Closed Dispute: `Dispute has been closed with status %1$s. See <a>dispute overview</a> for more details.`
Closed Inquiry: `Payment inquiry has been closed with status %1$s. See <a>payment status</a> for more details.`

#### Screenshots

_The following screenshots were captured prior to some minor copy tweaks._

Screenshot 2024-11-28 at 3 47 47 PM
Screenshot 2024-11-28 at 3 46 54 PM
Screenshot 2024-11-28 at 3 41 15 PM
Screenshot 2024-11-28 at 3 39 51 PM

-------------------

- [ ] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [ ] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `9828` or branch name `fix/9612-inquiry-order-note` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&woocommerce-payments-dev-tools&branches.woocommerce-payments=fix/9612-inquiry-order-note) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: b34e2996f375284d555e9e0e298518166480caaa
- Build time: 2024-12-02 21:32:00 UTC

_Note: the build is updated when a new commit is pushed to this PR._

File: includes/class-wc-payments-order-service.php [hunk 1]
```diff
@@ -1747,7 +1785,7 @@ private function compose_dispute_url( $charge_id ) {
 		return add_query_arg(
 			[
 				'page' => 'wc-admin',
-				'path' => '/payments/transactions/details',
+				'path' => rawurlencode( '/payments/transactions/details' ),
```
Comment (brucealdridge) [inline includes/class-wc-payments-order-service.php]: Not sure how this got reverted from https://github.com/Automattic/woocommerce-payments/blob/1137c625bd02502e003ebd8b09de473e82ea4b3a/includes/class-wc-payments-order-service.php#L1752 🤷

File: includes/class-wc-payments-order-service.php [hunk 2]
```diff
@@ -314,15 +314,17 @@ public function mark_order_blocked_for_fraud( $order, $intent_id, $intent_status
...
 	 * @return void
 	 */
-	public function mark_payment_dispute_created( $order, $charge_id, $amount, $reason, $due_by ) {
+	public function mark_payment_dispute_created( $order, $charge_id, $amount, $reason, $due_by, $status = '' ) {
 		if ( ! is_a( $order, 'WC_Order' ) ) {
 			return;
 		}
 
-		$note = $this->generate_dispute_created_note( $charge_id, $amount, $reason, $due_by );
+		$is_inquiry = strpos( $status, 'warning' ) === 0;
```
Comment (nagpai) [inline includes/class-wc-payments-order-service.php]: Nitpick ⛏️  - should we change 'warning' to 'warning_'  to make it similar to line 351?
```suggestion
		$is_inquiry = strpos( $status, 'warning_' ) === 0;
```

**Review by nagpai — APPROVED** [review]
Code changes look good to me. 

I have left a tiny optional nitpick. 

Manual test looks good too. This is good to merge! 🚢

**Pull request merged**